src/Controls/tests/Core.UnitTests/Layouts/GridLayoutTests.cs

```diff
@@ -184,5 +186,45 @@ public void ColumnDefinitionsGetBindingContext()
 
 			Assert.Equal(def2.BindingContext, context);
 		}
+
+		[Fact]
+		public async Task ColumnDefinitionDoesNotLeak()
+		{
+			// Long-lived column, like from a Style in App.Resources
+			var column = new ColumnDefinition();
+			WeakReference reference;
+
+			{
```

Comment:
I'm a bit confused by this syntax - what do the brackets do here? Should there be a condition or something else prefacing this code block?

Reply:
The `{ }` puts `var grid` out of scope at the end of the block, so you can no longer access `grid` after the closing brace. It's like a mini-method/function.

I might have been able to set it `null` instead, but this way is probably better.

Reply:
interesting, thanks for explaining!